app/static/scss/admin/users/users.scss

```diff
@@ -14,6 +14,73 @@
         margin-top: 1rem;
         display: block;
     }
+    @media screen and (min-width: 641px) {
+        .display-feature-table {
+            table, td, th {
+                overflow: hidden;
+                border: 1px solid #e0e0e0;
+            }
+            .highlighted {
```

Comment:
All the common styles between the two media queries should be brought into one common rule outside of a media queries.